### ISSUE TITLE
linux_net.py patch fails after upgrade to python-nova-2015.1.4

### ISSUE THREAD

**Issue by kamidzi** [bloomberg/chef-bcpc#1040]
```
[31mError executing action `apply` on resource 'bcpc_patch[nova-network-linux_net]'[0m
================================================================================[0m

[0mRuntimeError[0m
------------[0m
      errors raised by shasum both before and after:
[0m      BEFORE: shasum: WARNING: 1 computed checksum did NOT match
[0m
[0m      AFTER: shasum: WARNING: 1 computed checksum did NOT match
[0m
[0m
[0mCookbook Trace:[0m
---------------[0m
/var/chef/cache/cookbooks/bcpc/providers/patch.rb:76:in `block in class_from_file'[0m
```

**Comment by kamidzi** [comment]
As per #1041, this is more widespread in scope. For now: https://github.com/bloomberg/chef-bcpc/tree/bugfix/1040-b0rken-linux-net-patch

**Comment by erhudy** [comment]
I added another changeset on the broken patch branch that gates the Horizon Swift patch, since that issue was fixed upstream in 2015.1.4. Testing out now on existing broken build.
